core/renderers/block_rendering_rewrite/block_render_draw_highlight.js

```diff
@@ -108,40 +108,35 @@ Blockly.blockRendering.Highlighter.prototype.drawJaggedEdge_ = function(row) {
 
 Blockly.blockRendering.Highlighter.prototype.drawValueInput = function(row) {
   var input = row.getLastInput();
-  var steps = '';
   if (this.RTL_) {
     var belowTabHeight = row.height - input.connectionHeight;
 
-    steps =
-        Blockly.utils.svgPaths.moveTo(
-            row.xPos + row.width - this.highlightOffset_, row.yPos) +
-        this.puzzleTabPaths_.pathDown(this.RTL_) +
-        Blockly.utils.svgPaths.lineOnAxis('v', belowTabHeight);
+    this.steps_.push(Blockly.utils.svgPaths.moveTo(
```

Comment:
> Using svgPaths function and appending to this.steps_ rather than creating a local variable to follow patterns used in other functions in file.

I'm going to move us away from the steps array in this file soon.  Similar to https://github.com/google/blockly/pull/2817

Reply:
Got it, I will undo the changes that refactor to push onto the array.